dynamicprefixvici.c

```diff
@@ -227,174 +177,23 @@ static void ParseCommandLineArguments(int argc, char** argv, CommandLineArgument
         puts("pool_name and prefix_address are required, see -h for usage");
         exit(1);
     }
-
-    // If a provider prefix is provided or a slaid or a slasize, they to be all present
-    // sla_id may be zero
-    if(arguments->sla_size != 0  || arguments->prefix_size != 0)
-    {
-        if(arguments->sla_size == 0 && arguments->prefix_size == 0)
-        {
-            puts("sla_id, sla_size and prefix_size have to be present together");
-            exit(1);
-        }
-    }
 }
 
 static void Usage()
 {
-    printf("usage: dynamicprefixvici [-h] -n pool_name -p prefix_address [-s pool_size] [-l prefix_size] [-i sla_id] [-j sla_size]\n"
+    printf("usage: dynamicprefixvici [-h] -n pool_name -p prefix_address [-s pool_size[97]]\n"
```

Comment:
Is this how default values are shown in usage messages in other programs for example? If so OK.